benchmarks/kernels/benchmark_moe.py

```diff
@@ -17,7 +17,7 @@
 from vllm.platforms import current_platform
 from vllm.utils import FlexibleArgumentParser
 
-FP8_DTYPE = torch.float8_e4m3fnuz if current_platform.is_rocm(
+FP8_DTYPE = torch.float8_e4m3fnuz if current_platform.is_fp8_fnuz(
 ) else torch.float8_e4m3fn
```

Comment:
This appears in many places - could we just extract it into `current_platform.fp8_dtype()`?

Reply:
Great idea.  But I will also keep `is_fp8_fnuz()` because it is a useful shorthand before calling `normalize_e4m3fn_to_e4m3fnuz`.

Reply:
Yep, agreed 😃